lsp-protocol.el

```diff
@@ -397,9 +397,9 @@ See `-let' for a description of the destructuring mechanism."
 
 (lsp-interface (rls:Cmd (:args :binary :env :cwd) nil))
 
-(defconst lsp/rust-analyzer-inlay-hint-kind-type-hint "TypeHint")
-(defconst lsp/rust-analyzer-inlay-hint-kind-param-hint "ParameterHint")
-(defconst lsp/rust-analyzer-inlay-hint-kind-chaining-hint "ChainingHint")
+(defconst lsp/rust-analyzer-inlay-hint-kind-type-hint 1)
+(defconst lsp/rust-analyzer-inlay-hint-kind-param-hint 2)
+(defconst lsp/rust-analyzer-inlay-hint-kind-chaining-hint nil)
```

Comment:
This `nil` seems like it's a bug in rust-analyzer?  It feels like it should be `3` but it definitely isn't on my machine.

Reply:
Ah, so not a bug but an extension to the draft spec.

Reply:
Fixed in https://github.com/rust-analyzer/rust-analyzer/pull/11445